test/api_integration/apis/index_patterns/index_pattern_crud/update_index_pattern/main.ts

```diff
@@ -262,44 +262,6 @@ export default function ({ getService }: FtrProviderContext) {
       expect(response3.body.index_pattern.typeMeta).to.eql({ foo: 'baz' });
     });
 
-    it('can update index_pattern fields', async () => {
```

Comment:
have no idea how it worked before but if I'm not wrong users should not be able to change field names / type.